static/src/javascripts/projects/common/modules/identity/api.js

```diff
@@ -157,6 +157,7 @@ export const getUserEmailSignUps = (): Promise<any> => {
             url: (idApiRoot || '') + endpoint,
             type: 'jsonp',
             crossOrigin: true,
+            withCredentials: true,
```

Comment:
generally if we are authenticating with a cookie then I think we would hit `GET /useremails` rather than `GET /useremails/id` because the cookie contains the id.  (I realise that endpoint might not exist yet though).